composer.json

```diff
@@ -57,7 +57,7 @@
         "utopia-php/fetch": "0.2.*",
         "utopia-php/image": "0.6.*",
         "utopia-php/locale": "0.4.*",
-        "utopia-php/logger": "0.3.*",
+        "utopia-php/logger": "0.5.*",
         "utopia-php/messaging": "0.10.*",
         "utopia-php/migration": "0.4.*",
```

Comment:
is the migration library updated too ?

Reply:
Yeah, it's updated in the composer.lock

Reply:
<img width="904" alt="image" src="https://github.com/appwrite/appwrite/assets/26739219/08383bd2-0dca-4170-9a98-493b4f4f8a60">